Back-end/src/com/smartplanner/Term.java

```diff
@@ -0,0 +1,39 @@
+package com.smartplanner;
+
+import java.time.LocalTime;
+
+public class Term {
+    private int lengthInMin;
+    private int numberOfCycleDay; //date in cycle
```

Comment:
Wouldn't it be enough and more descriptive to name it "cycleDay" or "cycleDayNumber"?